documentation/ipi-install/modules/ipi-install-troubleshooting-cleaning-up-previous-installations.adoc

```diff
@@ -23,8 +23,10 @@ for i in $(sudo virsh list | tail -n +3 | grep bootstrap | awk {'print $2'});
 do
   sudo virsh destroy $i;
   sudo virsh undefine $i;
-  sudo virsh vol-delete $i --pool default;
+  sudo virsh vol-delete $i --pool $i;
```

Comment:
Same as above, we're removing $i, but the pool is default, isn't it?

Reply:
Default is the name, technically can be named anything. For simplicity, I think we leave default as the name.

Reply:
Same conditionals here (check output from kni4)